index.js

```diff
@@ -42,30 +44,34 @@ class Bumper extends Plugin {
     if (!out) return;
     return Promise.all(
       castArray(out).map(async out => {
-        const { file, type, path } = parseFileOption(out);
+        const { file, type, path, glob } = parseFileOption(out);
 
         this.log.exec(`Writing version to ${file}`, isDryRun);
 
         if (isDryRun) return noop;
 
-        if (type === 'application/json') {
-          const data = await readFile(file, 'utf8').catch(() => '{}');
-          const indent = detectIndent(data).indent || '  ';
-          const parsed = JSON.parse(data);
-          set(parsed, path, version);
-          return writeFile(file, JSON.stringify(parsed, null, indent) + '\n');
-        } else if (type === 'text/yaml' || type === 'application/x-yaml') {
-          const data = await readFile(file, 'utf8').catch(() => '{}');
-          const indent = detectIndent(data).indent || '  ';
-          const parsed = yaml.safeLoad(data);
-          set(parsed, path, version);
-          return writeFile(file, yaml.safeDump(parsed, { indent: indent.length }) + '\n');
-        } else if (type.startsWith('text/')) {
-          const { latestVersion } = this.config.contextOptions;
-          const read = await readFile(file, 'utf8').catch(() => latestVersion);
-          const versionMatch = new RegExp((latestVersion || '').replace(/[.*+?^${}()|[\]\\]/g, '\\$&'), 'g');
-          const write = read ? read.replace(versionMatch, version) : version;
-          return writeFile(file, write);
+        const files = glob ? await fg(file, { dot: true }) : [file];
```

Comment:
Why not just always assume it's a glob pattern? I guess a "real" non-glob filename will also just work, right?